style.css

```diff
@@ -0,0 +1,65 @@
+@import url("https://fonts.googleapis.com/css2?family=Source+Sans+Pro:ital,wght@0,300;0,400;0,600;0,700;0,900;1,600;1,700;1,900&display=swap");
+@import url('https://fonts.googleapis.com/css2?family=Poppins&display=swap');
+
+body {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+  background-color: #e5e5e5;
+}
+
+#bg-Img {
+  background-image: url(images/bg-image.png);
+  background-repeat: no-repeat;
+  height: 588px;
+}
+
+.main {
+  padding: 40px 24px 40px 24px;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  padding: 58px 24px 58px 38px;
+}
+
+.heading h1 {
+  font-family: "Poppins", sans-serif;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  color: #172b4d;
+}
+
+.heading p {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #344563;
+}
```

Comment:
The new updates you made has created an error in the background color of your design since it is now different to the actual design as shown below. Kindly check and fix this. 

### Your design 

![Screenshot from 2023-03-25 22-18-26](https://user-images.githubusercontent.com/65233860/227745056-f2b61b5c-2113-4ab7-abd6-258e2746393e.png)



### Actual design 


![Screenshot from 2023-03-25 22-20-58](https://user-images.githubusercontent.com/65233860/227745062-5ede40d7-e834-42d9-9352-42eab9ef72c0.png)